frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -34,16 +34,32 @@ class Recipe extends Component {
         difficulty: response.data.difficulty,
         duration: response.data.duration,
         servings: response.data.servings,
+        userId: response.data.user_id,
       });
     });
   }
 
   render() {
     const { user, setJwt } = this.props;
     const {
-      directions, title, ingredients, snippet, difficulty, duration, servings, photo,
+      directions, title, ingredients, snippet, difficulty, duration, servings, photo, userId,
     } = this.state;
 
+    const renderButtons = function () {
```

Comment:
I think keeping this as an arrow function the way you had it before is preferable, especially to maintain consistency with all of the other functions. The linter was complaining because the function is just pure JSX, so you skip the return call and just wrap the body in parentheses. So it can just be:

```
const renderButtons = () => (
  // JSX here
);
```

Alternatively, since it's not doing anything dynamic, you could just convert it from a function to a JSX string by removing the `() =>` bit, and just use `renderButtons` without the parentheses on line 88.